### PR TITLE
Fix session migration and assign message history to correct sessions

### PR DESCRIPTION
**Pull request by Blackoverflow**
This one contains:
- a migration for the missing sessions
- a migration to correctly assign the message history to the new sessions (leaving the old sid in a new column sv1id)
- a update of signal-golang/textsecure to version 1.19.0

Still to do:

- [x]  cleanup
- [ ]  verification that this works like intended
- [x] reference new release of textsecure

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -1,6 +1,10 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 github.com/Blackoverflow/gcmsiv v0.0.0-20201031155123-20af2af977ff h1:l3WopnX/gWnDItXZdr2Ygq0+4ZwXCSyozyPNR8jdb8I=
 github.com/Blackoverflow/gcmsiv v0.0.0-20201031155123-20af2af977ff/go.mod h1:t1lmwOv+IYxaUkr2qIKXEfn90t4GgocS0qy8jkDonbg=
+github.com/Blackoverflow/textsecure v1.4.3-0.20221028183342-5dd575a18460 h1:ZPUbXWULhddzbWI9AnpKdbKGY317FatHsAyTQzJg3Xs=
```
Comment (olof-nord) [inline go.sum:4]: Any reason why we are not using the https://github.com/signal-golang/textsecure package anymore?

**Review by olof-nord — COMMENTED** [review]
I know it is work in progress, but if we are thinking of switching to a fork of textsecure lets clarify the reasons why and start to think of how to upstream our changes?

File: go.sum (hunk 1 above)
Comment (olof-nord) [reply go.sum:4]: Makes sense. I opened an issue in textsecure as a reminder 👍

Comment (Blackoverflow) [reply go.sum]: I have now changed this by referencing the currently last commit of signal-golang/textsecure.
This still needs to be replaced after a new version is released, but at least my fork isn't referenced anymore.

**Comment by nuehm-arno** [comment]
Will do a review tomorrow!

**Comment by nuehm-arno** [comment]
I tried to review, but the changes are a bit too much for my limited go knowledge - sorry.

**Pull request merged**